components/bl-transferList-component/src/transfer-buttons.js

```diff
@@ -0,0 +1,131 @@
+export const TransferButtons = props => {
+  const {
+    enableMoveAll,
+    allSelected,
+    left,
+    right,
+    leftSelected,
+    rightSelected,
+    setAllSelected,
+    setLeft,
+    setRight,
+    eventHandlers,
+  } = props;
+
+  const { onChange } = eventHandlers;
+
+  const moveAllToRight = () => {
```

Comment:
Could we create an custom hook for all this handlers?

Something like `useMoveHandlers`

Reply:
Of course we can, but I have a question, as we just put all the handlers in a separate file and do not use any hooks there, so it's seems more like extracting functions than a custom hook. Am I understanding you correctly?